Chapters/4-6-Chapter28.md

```diff
@@ -0,0 +1,201 @@
+
+The oars cut back to a silence, but no response came.
+
+"You know them?" I asked.
+
+"I know that voice, I heard it often when I was stowed away on Ramne's ship."
+
+"How do you know we can trust them?"
+
+"There's a reason Ramne had to steal a boat."
+
+"Shinna!" thi yelled again. "It's Ginnu!"
+
+The water remained silent for a moment more, then began with quiet disquiet, rummaging through the water. The sound followed a path toward the shore some fifty paces toward Rudjan. I looked to Annu who looked back at me with hae hand extended in a gesture of fellowship. I took it in mine, and together we walked along the shoreline toward the place where they were about to land.
+
+Three boats slid to a stop upon the sandy sea floor, their crew sitting forward on the benches, sprawled out upon their oarshafts to relieve their muscles. A maman hopped down from the back of the middle boat with a rope and lantern, and waded toward the land trying to make out anything identifiable.
+
+"Is anyone there?" thi called to the darkness, "Ginnu?"
+
+Annu waited for a moment, observing hae with caution. Thi opened up once thi felt ready, "Shinna, over here!"
+
+The maman paused and looked our way, hae eye catching the both of us.
+
+"Ginnu! is that really you?" thi exclaimed. Annu and I waded out to meet hae voice, while Shinna waded as far as hae rope would allow.
+
+"Melna told me what happened," said Shinna. There was a guilt about hae voice. "I didn't believe what she said until I saw the evidence."
+
+"I owe her everything."
+
+Shinna turned hae sight toward me, and then toward something behind me. I turned to see Laen and Boon walking toward us.
+
+"Looks like you've met some friends."
+
+"Linyu and these saffas have been throught a lot," thi said with an air of endearment. "But I'm glad they found me."
+
+"Let me find a rock 'n we can talk more on land."
+
+Thi found a sturdy rock to bind the rope to and with a few flicks of the wrist thi commuted a series of loops along the length of the rope, pulling it to and fro. As if from nowhere, a wide knot encircled the rock and held fast. I gasped in awe while Annu pulled me from my trance out of the cold water.
+
+"Ginnu," Shinna began awkwardly, "I... I'm sorry I never checked on you."
+
+"Don't," thi swallowed hae emotions. "It exists only in the past. Leave it there."
+
+"He promised us that you were a friend. That you wanted to remain discreet."
+
+"I can't fault you for things you didn't know."
+
+"It's my duty to know what happens on my ship."
+
+"Shinna, you can sulk about it all you want, but the past is the past. I'm done with that man. and I'd like it to stay that way."
+
+Thi bowed hae head, hae guilt defeated.
+
+Annu continued, "That name: Ginnu. It's what Ramne called me, but it's not who I am. My name is Annu."
+
+"Annu... I won't forget it," thi said solemnly. "Annu and Linyu. Mamen of the dark."
+
```

Comment:
So the message of this entire story: the people who are nurturing are the best of 'em all!